tests/unit/testplan/common/utils/test_match.py

```diff
@@ -152,6 +152,73 @@ def test_match_all(self, basic_logfile):
         assert matches[0].group(0) == "fourth"
         assert matches[1].group(0) == "fifth"
 
+    def test_not_match(self, basic_logfile):
```

Comment:
there is already a ``test_not_match`` in this file, did I miss something ?

Reply:
Good catch! Not sure what happened there.